target/linux/x86/base-files/lib/preinit/43_grow_rootfspart

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
```

Comment:
Preinit is sourced (/lib !), please remove shebang and execute bit on file

Reply:
It was copied from another preinit file:

```
root@(none):/lib/preinit# head -1 * | grep '#!/bin/sh' | wc -l
13
```

Maybe they all should be removed.

Done

Reply:
I did that a while ago but probably did not hit them all ...
https://github.com/openwrt/openwrt/commit/48c1fdd046f409fd5e28932f1d53343fa214fbc4